src/librustc_traits/chalk/db.rs

```diff
@@ -0,0 +1,479 @@
+use rustc_middle::traits::{ChalkRustDefId as RustDefId, ChalkRustInterner as RustInterner};
```

Comment:
probalby would be good to have a module level comment here just giving a high-level view of what's going on in this file

Reply:
Added. It's pretty short, but something.